adobe/react-spectrum
# fix: port placeholder change from spectrum-css

Closes #2759 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- Run any page using picker
- Check the color of the text used as the placeholder
- It should be gray-800 colors

## 🧢 Your Project:
